base_tier_validation_definition_server_action/models/tier_definition.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 ForgeFlow S.L. (http://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class TierDefinition(models.Model):
+    _inherit = "tier.definition"
+
+    definition_type = fields.Selection(
+        selection_add=[("server_action", "Server Action")]
```

Comment:
When using `selection_add`, you should add attribute `ondelete={"server_action": "cascade"}` or `ondelete={"server_action": "set default"}`